crates/sovereign-sdk/rollup-interface/src/state_machine/soft_confirmation.rs

```diff
@@ -282,3 +283,40 @@ impl<'txs, Tx: Clone> SignedSoftConfirmation<'txs, Tx> {
         self.timestamp
     }
 }
+
+/// Signed version of the `UnsignedSoftConfirmation` used in Genesis
+/// Contains the signature and public key of the sequencer
+#[derive(PartialEq, Eq, BorshDeserialize, BorshSerialize, Serialize, Deserialize)]
+pub struct SignedSoftConfirmationV1<Tx> {
+    l2_height: u64,
+    hash: [u8; 32],
+    prev_hash: [u8; 32],
+    da_slot_height: u64,
+    da_slot_hash: [u8; 32],
+    da_slot_txs_commitment: [u8; 32],
+    l1_fee_rate: u128,
+    txs: Vec<Tx>,
```

Comment:
I think there's a misunderstanding here, instead of txs we would have blobs: Vec<u8> here

Reply:
If we are not forgetting about any other breaking change, I think this is the only thing that makes test fail rn :)